components/features.tsx

```diff
@@ -1,322 +1,82 @@
-                  d="M30 39.313l-4.18 2.197L27 34.628l-5-4.874 6.91-1.004L32 22.49l3.09 6.26L42 29.754l-3 2.924"
-                  strokeLinecap="square"
-                  strokeWidth="2"
-                  fill="none"
-                  fillRule="evenodd"
-                />
-                <path
-                  className="stroke-current text-white-main"
-                  d="M43 42h-9M43 37h-9"
-                  strokeLinecap="square"
-                  strokeWidth="2"
-                />
-              </svg>
-              <h4 className="h4 mb-2">Knows your code.</h4>
-              <p className="text-center text-lg text-gray-600">
-                Pear has context on your codebase so you can ask questions about
-                your code.
-              </p>
-            </div>
 
-            {/* 2nd item */}
-            <div
-              className="relative flex flex-col items-center"
-              data-aos="fade-up"
-              data-aos-delay="100"
-              data-aos-anchor="[data-aos-id-blocks]"
-            >
-              <svg
-                className="mb-4 h-16 w-16"
-                viewBox="0 0 64 64"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <circle
-                  className="fill-current text-primary-500"
-                  cx="32"
-                  cy="32"
-                  r="32"
-                />
-                <path
-                  className="stroke-current text-white-main"
-                  strokeWidth="2"
-                  strokeLinecap="square"
-                  d="M21 23h22v18H21z"
-                  fill="none"
-                  fillRule="evenodd"
-                />
-                <path
-                  className="stroke-current text-white-main"
-                  d="M26 28h12M26 32h12M26 36h5"
-                  strokeWidth="2"
-                  strokeLinecap="square"
-                />
-              </svg>
-              <h4 className="h4 mb-2">Auto-Implement Features</h4>
-              <p className="text-center text-lg text-gray-600">
-                Ask for a feature and receive a PR implementing it.
-              </p>
-            </div>
+    {
+      header: "No more tedious changes,\nor forgetting language\nsyntax",
+      description: "Directly make changes inline by pressing CMD+I (ALT+L on Windows), and choose what you want to keep. \n\n Here, we ask Pear to help us handle edge cases 😏",
+      imageSource: "https://s3-alpha-sig.figma.com/img/9229/8c19/f69e8c788124e01751223a500df85aea?Expires=1719792000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=nBte2jUC56Kr11M1WApQo-LQyYWAIYyxrmSVNyOgOZKMPAfN0Wem7dACcXkpI8T-ytGA~pA-CHwyZ9fxKAlD88kBI1ZAgFmDEDCsIhuBBOVYtc0XyPkC2S0R~yQAqgUUMUqG8Si34HvSDfq4N9x1vcZi3osCfpbPY7vFKJCze-E538Hw-7aA4wWAC1h-MLnPM0XvMZoFwPwBXB95ogiKaG8qGfFlSegAnupb2sqR2TmfLse6dG5jqk-dOpzUF3zbnH-9Q-nkJK9Fmdw~LHvj4z3wcpN75bWpNSgzyuXR3GXwJPxamvxs9Af9VNj8l2DCS~p4YLe-C1iouGrkV-h-yw__"
```

Comment:
Sorry should've sent this earlier. The original gifs can be downloaded here: https://drive.google.com/drive/u/0/folders/19rOYS_rbuBL4YDfIS3ZxKbUWZNXxR0-7

Reply:
Thank you